recipes/rai-core-flask/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "rai-core-flask" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/rai_core_flask-{{ version }}.tar.gz
+  sha256: 5498b8c6beabd690d58d4038e833f62e92976e9fc55dba9c319d483d41bf4585
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
```

try this:

```suggestion
  script: touch requirements.txt && {{ PYTHON }} -m pip install . -vv
```

and delete the `requirements.txt` from the directory.

the fact that the `requirements.txt` is empty should not matter.

---

done 🤞🏻